addons/mail/static/src/scss/composer.scss

```diff
@@ -62,8 +62,11 @@
 }
 
 .o_mail_composer_form {
```

Comment:
note that this class is also used in `survey` and in `website_slides`, are we sure it won't break those view ? (the `width: 100%;`)

Reply:
Yeah I just checked they both aim to look like a regular composer and it doesn't bother me at least. I'll make a mental note to check these things in advance though.